metrics/prommetrics.py

```diff
@@ -2,12 +2,18 @@
 import os
 import sys
 import dateparser
+import logging
 
 from rx import Observable
 
 #Prometheus connection stuff
 from prometheus import Prometheus
 
+# Set up logging
+_LOGGER = logging.getLogger(__name__)
+# Log record format
+logging.basicConfig(format='%(asctime)s:%(levelname)s: %(message)s', level=logging.INFO)
```

Comment:
lets setup the logger in `app.py` like in https://github.com/AICoE/log-anomaly-detector/commit/c0328cc940dc2de1ae53d66ab0bbe89b0623adb5